### ISSUE TITLE
The email address is probably wrong

### ISSUE THREAD

**Issue by wilfwilson** [gapdays/gapdays2025-spring#5]
I'm guessing that this line needs to be updated:

https://github.com/gapdays/gapdays2025-spring/blob/ffaf7d05ddc83b324d2961aef9690e3c8762361c/_config.yml#L15C19-L15C48
```
email:            gapdays2024-summer@gapdays.de
```

But only, of course, once the email forwarder has actually been created.

**Comment by fingolfin** [comment]
Indeed. @jdebeule if you want I can create this as an alias which forwards to a set of email addresses of your choice (e.g. your own, possibly a secretary for handling registrations etc.)
